### DISCUSSION TITLE
Does Photoprism Send My Photo Data to the Internet?

### DISCUSSION THREAD

**Discussion by rvrosm** [photoprism/photoprism#4134] [Q&A] [answered]
Since Photoprism is AI-Powered, does it mean my photos' data gets sent over the internet? I don't have AI on my local machine, so I assume it needs internet access for the AI magic. 

And if it doesn't send data to some API, how is it "AI-Powered"?

**Comment by lastzero** [comment, marked as answer]
PhotoPrism performs all AI locally, so an Internet connection is only needed for fetching geodata, loading maps, and getting updates. See our FAQ and Documentation for details:
https://docs.photoprism.app/user-guide/faq/

**Reply by rvrosm** [reply]
Thank you.
